### ISSUE TITLE
Command to toggle a window's visibility (akin to clicking on the icon)

### ISSUE THREAD

**Issue by ficcdaf** [user-none/KDocker#140]
**REQUESTS FOR THE FOLLOWING WILL NOT BE CONSIDERED**
- [Wayland support](https://github.com/user-none/KDocker/wiki/Wayland)
- [Close to tray](https://github.com/user-none/KDocker/wiki/Close-to-Tray)

---

**Is your feature request related to a problem? Please describe.**
Basically it would be great to be able to toggle the visibility of a docked window with a keybind. So the same as clicking on its icon in the tray, but with the keyboard instead.

**Describe the solution you'd like**
I would like a CLI option that lets us specify a certain window or application that is currently docked, and lets us either toggle it or set it to "visible" or "hidden". The CLI option can give flexiblity to the users in deciding how they want to trigger this functionality, such as a keybinding in the window manager that will trigger this command.

**Describe alternatives you've considered**
Kdocker could also listen for the keybind directly, but I feel this would be simpler and give users more options in customization. Personally I prefer having all my keybindings in one place (my i3 config file), so this would also be good for power users. 

**Additional context**
Basically, I want to be able to toggle Spotify without needing to click on the icon with my mouse, and a keybind would be amazing for this. Please forgive me if this functionality already exists, I've tried reading the manual and I can't seem to find anything.

**Comment by user-none** [comment]
KDocker can't listen for keybindings unfortunately. So you would need to handle this via the window manager. I don't know about i3 specifically. You'll need to figure out how to call CLI command with it.

You can interact with KDocker using DBus messages. There are a number of [actions](https://github.com/user-none/KDocker?tab=readme-ov-file#docked-window-management) you can tell KDocker to take with docked windows. The README includes [examples](https://github.com/user-none/KDocker?tab=readme-ov-file#cli-examples) for sending DBus messages using the CLI.

You'd use the `listWindows` to find the window id for the docked Thunderbird (it changes every time Thunderbird restarts). Then use `showWindow` with the window id you found. The `hideWindow` function takes the same window id. You'll probably need put this into its own script because of the two steps, lookup id, and use id.

**Comment by ficcdaf** [comment]
Thank you, this is exactly the information I was looking for. Cheers!
